objects/src/testing/storage.rs

```diff
@@ -33,21 +33,24 @@ impl AccountStorageBuilder {
     pub fn new() -> Self {
         Self {
             slots: vec![
-                AccountStorage::mock_item_2().slot,
                 AccountStorage::mock_item_0().slot,
                 AccountStorage::mock_item_1().slot,
+                AccountStorage::mock_item_2().slot,
```

Comment:
As mentioned in the other comment, I[ think these should be removed again](https://github.com/0xPolygonMiden/miden-base/pull/871/files#diff-b85fb1019997da2501d1ab4d7877f1e83e29b0a20329eaf4c828026a59ff3a38L31-L40).